content/en/docs/reference/glossary/replication-controller.md

```diff
@@ -1,19 +1,26 @@
 ---
-title: Replication Controller
+title: ReplicationController
 id: replication-controller
 date: 2018-04-12
 full_link: 
 short_description: >
-  Kubernetes service that ensures a specific number of instances of a pod are always running.
+  Ensures a specific number of instances of a Pod are running.
 
 aka: 
 tags:
 - workload
 - core-object
 ---
- Kubernetes service that ensures a specific number of instances of a pod are always running.
+ A (deprecated) workload resource that manages a replicated application, ensuring that
+a specific number of instances of a Pod are running.
 
-<!--more--> 
+<!--more-->
 
-Will automatically add or remove running instances of a pod, based on a set value for that pod. Allows the pod to return to the defined number of instances if pods are deleted or if too many are started by mistake.
+ReplicationController automatically adds or remove running instances of a Pod, based on a set value.
+The control plane makes changes to ensure the defined number of instances are running, even if the
+ReplicationController's Pods are deleted or if too many are started by mistake.
 
+{{< note >}}
+ReplicationController is deprecated. See {{< glossary_tooltip text="Deployment" term_id="deployment" >}}
+for the resource that replaced it.
```

Comment:
Refer to: https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/#replicationcontroller

The counterpart for ReplicationController is ReplicaSet, not Deployment.

Reply:
@tengqm If you're a developer thinking of creating a ReplicationController, you should use a Deployment.
(and if you read about Deployment it will tell you about ReplicaSet anyway).

See https://kubernetes.io/docs/concepts/workloads/controllers/replicationcontroller/

Reply:
@sftim We can recommend users to use Deployment but saying that "see Deployment for the resource that replaced it" is not accurate. `ReplicaSet` has its own page, by the way. https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/